src/delimiter.js

```diff
@@ -329,14 +329,12 @@ const sqrtSvg = function(sqrtName, height, viewBoxHeight, options) {
     }
     const pathNode = new domTree.pathNode(sqrtName, alternate);
 
-    let attributes = [["width", "100%"], ["height", height + "em"]];
+    const attributes = [["width", "400em"], ["height", height + "em"]];
```

Comment:
Why does this have to be `400em` as opposed to `100%`?

Reply:
I'll give `100%` a try, but I don't think it will work. I'm now defining the `viewBox` dimensions in the same SVG as the width attribute. The height has a factor of 1000, so I think the width needs consistent treatment.

Reply:
I was wrong. It does work to use `width='100%'`. Not only that but when `width='100%'`, I can remove the CSS `oveflow:hidden` and it still works. So consider that done.

I wonder if that will cause sub-pixel changes in all the screenshots.

Reply:
I retract my last statement. When I use `width='100%`, IE does not slice off the long tail. I going back to `width='400em'` and CSS `overflow:hidden`.

Reply:
Final word. It turns out that it doesn't matter if I write `width='400em'` or `width='100%'`. It would matter, except that IE will not work unless I include CSS `overflow:hidden`. Once I include the CSS condition, both widths work. I'm going with 400em.